src/test/java/teammates/test/cases/util/AdminLogQueryTest.java

```diff
@@ -32,7 +33,7 @@ public void testAdminLogQuery() {
         ______TS("Test setTimePeriod");
         query = new AdminLogQuery(versionList, null, null);
         assertEquals(0, query.getStartTime());
-        assertTrue(endTime != query.getStartTime());
+        AssertJUnit.assertNotSame(endTime, query.getStartTime());
```

Comment:
`assertNotSame` checks that the two objects don't refer to the same Object, not checking for inequality. If you want to fix this issue, the correct assertion to use is `assertNotEquals`, and you should define that method in `BaseTestCase` as what have been done for all the used assertion methods.

Reply:
Okay.